### PR TITLE
chore(docs): Amending insert data page with info on COPY

### PR DESCRIPTION
**Pull request by amyshwang**
1. Changing the order of the list.
2. Adding info to reflect changes brought in by COPY.

### CODE REVIEW

File: docs/develop/insert-data.md [hunk 1]
```diff
@@ -622,8 +605,8 @@ See:
 ### Constructing well-formed messages
```
[the remark this answers is not in the document]
Comment (amunra) [reply docs/develop/insert-data.md:605]: Also, later in the document "From version 6.3, QuestDB will disconnect on the first error encountered on a
given TCP ILP connection." .. I think we can remove references to version 6.3 as disconnect-on-error has been around for a little while now.

File: docs/develop/insert-data.md [hunk 2]
```diff
@@ -706,7 +689,7 @@ is documented [here](/docs/reference/api/ilp/authenticate).
 
 ### Third-party Library Compatibility
 
-Use our own client libraries and/or protocol documentation: Clients intended to
+Use our own [client libraries](/docs/reference/clients/overview) and/or protocol documentation: Clients intended to
 work with InfluxDB will not work with QuestDB.
 
 ## PostgreSQL wire protocol
```
[the remark this answers is not in the document]
Comment (amunra) [reply docs/develop/insert-data.md:695]: The Python example later on mentions that `psycopg3` does not support bind variables yet.
This information is out of date now as it does: https://www.psycopg.org/psycopg3/docs/basic/params.html

The example should be updated (maybe Javier can help?).

Related: I suspect we'd need up update the equivalent example in the `saas-client` project for the cloud documentation.

File: docs/develop/insert-data.md [hunk 3]
```diff
@@ -10,19 +10,14 @@ languages and tools.
 ingestion method in QuestDB and is recommended for high-performance
 applications.
 
+For transactional data inserts, use the [PostgreSQL wire protocol](#postgresql-wire-protocol).
+
 For operational (ad-hoc) data ingestion the [Web Console](#web-console) makes it
```
[the remark this answers is not in the document]
Comment (tris0laris) [reply docs/develop/insert-data.md:15]: comma works

File: docs/develop/insert-data.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (amyshwang) [reply docs/develop/insert-data.md:605]: Extracted to https://github.com/questdb/questdb.io/issues/1001

File: docs/develop/insert-data.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (amyshwang) [reply docs/develop/insert-data.md:695]: "PostgreSQL wire protocol" - I added some wording from the PR yesterday.
Python - extracted to https://github.com/questdb/questdb.io/issues/999

**Review by amunra — APPROVED** [review]
Lookin' good!

**Pull request merged**